src/Illuminate/Session/Middleware/StartSession.php

```diff
@@ -41,9 +41,11 @@ public function __construct(SessionManager $manager)
     /**
      * Handle an incoming request.
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  \Closure  $next
```

Comment:
You may add the removed spaces here.

Reply:
What is the double space for?
I used phpstorm formatting, it is he who deleted this space. If necessary, I will actually re-apply that.